src/components/chip-set/chip-set.tsx

```diff
@@ -118,7 +118,7 @@ export class ChipSet {
     /**
      * For chip-set of type `input`. Sets delimiters between chips.
      */
-    @Prop({ reflect: true })
+    @Prop()
```

Comment:
This is a breaking change. Please remove this commit.

Reply:
ok, I will remove it